src/index.html

```diff
@@ -1,17 +1,153 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Hello</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
   </body>
+  <main>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="form-field">
```

Comment:
`<fieldset class="form-field">` sounds confusing.
Here's definition of fieldset. Can it really have class `form-field`?

<img width="722" alt="Screenshot 2023-04-19 at 14 51 13" src="https://user-images.githubusercontent.com/103033758/233066268-8ca76a5c-4a1f-4dd4-83f8-0e319215835a.png">

Reply:
Thank you a lot for comments!
The next time, please pay attention that in the code below I deleted a class, so I have only 2 atributes

<form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
    method="post">

![image](https://user-images.githubusercontent.com/88103621/233072115-f642d788-19d9-42ff-ba62-2deb50e9a889.png)

Reply:
@RoksolanaTylipska 
> The next time, please pay attention that in the code below I deleted a class, so I have only 2 atributes

yeah, I've seen that. Let me correct my comment a little:

If the names of attributes are too long - move **each** of them to a new line. Even if you have only 2 of them - both should be on a separate line

**Context**:
When you write code one of your TOP priorities is to keep it clear for those who read the code (likely that they will work with it further). So you should pay attention to code readability, so it would be easier to work with.